src/controller/seat.ts

```diff
@@ -42,13 +41,12 @@ export const checkCurrentSeat = async (
   res: express.Response
 ) => {
   try {
-    const { JWT } = req.body;
-    const payload = <JwtPayload>decryptJwt(JWT);
+    const { id } = res.locals;
```

Comment:
마이너하지만
```suggestion
    const { id: giverId } = res.locals;
```
도 좋을 것 같슴다